wp-content/themes/twentyseventeen-wp20/functions.php

```diff
@@ -8,13 +8,16 @@
 add_action( 'wp_enqueue_scripts',    __NAMESPACE__ . '\enqueue_scripts', 11              );
 add_filter( 'get_custom_logo',       __NAMESPACE__ . '\set_custom_logo'                  );
 add_filter( 'body_class',            __NAMESPACE__ . '\add_body_classes'                 );
+add_filter( 'document_title_separator', __NAMESPACE__ . '\set_document_title_separator'  );
```

Comment:
Indentation is out here. Personally I don't find it a big deal, but since everything else is aligned...

Reply:
Also neither of these functions run for me when Yoast SEO is enabled